hazelcast/src/main/java/com/hazelcast/spi/impl/InternalOperationService.java

```diff
@@ -35,15 +34,13 @@
  */
 public interface InternalOperationService extends OperationService {
 
-    void onMemberLeft(MemberImpl member);
```

Comment:
why is this no longer a part of InternalOperationService? 
it's not a lifecycle event, it's rather a business event.